src/Horse.Jhonson.pas

```diff
@@ -35,7 +35,7 @@ procedure Jhonson(Req: THorseRequest; Res: THorseResponse; Next: TProc);
     LWebResponse.Content := TJSONValue(LContent).ToJSON;
     LWebResponse.ContentType := 'application/json';
   end
-  else if not Assigned(LContent) then
+  else 
```

Comment:
There is another observation. If you pass a stream, you will get 204 depending on the order of execution. It is necessary to check if the content is of type "json"...

Reply:
Ok... Seria adicionar essa verificação LContent.InheritsFrom(TJSONValue)?

Reply:
It´s can also be... but I believe you can simplify it even more, just checking the type once!

example:

```pascal
    if LContent.InheritsFrom(TJSONValue) then
    begin
      if Assigned(LContent) then
        LWebResponse.Content := TJSONValue(LContent).ToJSON
      else
        LWebResponse.StatusCode := THttpStatus.NoContent.ToInteger;
      LWebResponse.ContentType := 'application/json';
    end;
```

Reply:
Nice man, it's great code. Thanks

Reply:
If object is nil, application raise exception on verification :/

Reply:
Isso eu faria a parte na minha rotina, não teria como implementar para que o horse me devolvesse essa codigo ?